### PR TITLE
Upgraded version to 1.1

### PR DESCRIPTION
**Pull request by peter-tackage**
Added wrapper method to return a token using username/password
credentials.

### CODE REVIEW

File: src/main/java/com/readmill/api/ReadmillWrapper.java [hunk 1]
```diff
@@ -207,6 +207,69 @@ public Token obtainTokenOrThrow(String authorizationCode) throws IOException, JS
...
+  
+  /**
+   * Request an OAuth2 token from Readmill
+   * @param  request the token request
+   * @return the token
+   * @throws java.io.IOException network error
+   * @throws com.ReadmillAPI.api.CloudAPI.InvalidTokenException unauthorized
+   * @throws com.ReadmillAPI.api.CloudAPI.ApiResponseException http error
+   */
+//  protected Token requestToken(Request request) throws IOException {
```
Comment (christoffer) [inline src/main/java/com/readmill/api/ReadmillWrapper.java]: Why is this commented out? By mistake?

File: src/main/java/com/readmill/api/ReadmillWrapper.java [hunk 2]
```diff
@@ -207,6 +207,69 @@ public Token obtainTokenOrThrow(String authorizationCode) throws IOException, JS
     JSONObject tokenJson = new JSONObject(tokenResponse);
     return new Token(tokenJson);
   }
+  
+  /**
+   * Returns a token using direct username and password credentials.
+   * 
+   * Returns The token granted or null if the credentials were declined.
+   * 
+   */
```
Comment (christoffer) [inline src/main/java/com/readmill/api/ReadmillWrapper.java:223]: Would be amazing if this could follow the same javadoc form as all the other methods =)

**Comment by christoffer** [comment]
Cool stuff, Peter!

It would be cool if you could get the whitespace the same as all the existing code before mergin (2 spaces indents + no new lines after or before function curlies {).

Like this:

``` java
void func() {
  code();
}
```

Thanks!

File: src/main/java/com/readmill/api/ReadmillWrapper.java (hunk 1 above)
Comment (peter-tackage) [inline src/main/java/com/readmill/api/ReadmillWrapper.java]: I'll remove it. I misread the diffs when committing and thought it was something you had left in there.

File: src/main/java/com/readmill/api/ReadmillWrapper.java (hunk 2 above)
Comment (peter-tackage) [inline src/main/java/com/readmill/api/ReadmillWrapper.java:223]: Now corrected.

**Comment by peter-tackage** [comment]
The two commits should address your comments.

**Comment by christoffer** [comment]
Amazing!

Wonder why it can't automatically be merged.

Could you try to do 

``` bash
git checkout master
git pull
git checkout username-password
git merge master
git push
```

**Comment by peter-tackage** [comment]
It looks like I hadn't pulled your v1.0 changes. Hmm, I think I may have merged from my master clone...

**Comment by peter-tackage** [comment]
That is, merged without including the two extra commits that I had made in the username-password branch. It looks like it can be automatically merged now. Perhaps you give it a go? I'm a menace to society!

**Comment by peter-tackage** [comment]
Ok, I think it's good now. Although, I noted that there are no v1.1 snapshots and the README.md Maven dependency text are still referencing v1.0 - no biggie. Perhaps something to do in your spare time ;-)

**Pull request merged**